MAC上使用xx-net，出现以下错误，救助！openssl应该没装错啊

Apr 02 13:51:26.111 - [INFO] start XX-Net get_version_fail
Apr 02 13:51:26.112 - [DEBUG] start confirm_xxnet_exit
Apr 02 13:51:26.114 - [DEBUG] good, xxnet:8087 cleared!
Apr 02 13:51:26.116 - [DEBUG] good, xxnet:8085 clear!
Apr 02 13:51:26.116 - [DEBUG] finished confirm_xxnet_exit
Apr 02 13:51:26.133 - [ERROR] start module gae_proxy fail:dlopen(/Users/apple/anaconda/lib/python2.7/site-packages/OpenSSL/rand.so, 2): Library not loaded: libssl.1.0.0.dylib
  Referenced from: /Users/apple/anaconda/lib/python2.7/site-packages/OpenSSL/rand.so
  Reason: image not found
Apr 02 13:51:26.135 - [ERROR] Except stack:Traceback (most recent call last):
  File "/Users/apple/Downloads/XX-Net-2.9.6/launcher/module_init.py", line 37, in start
    proc_handler[module]["imp"] = __import__(module, globals(), locals(), ['local', 'start'], -1)
  File "/Users/apple/Downloads/XX-Net-2.9.6/gae_proxy/start.py", line 14, in <module>
    import local.proxy as client
  File "/Users/apple/Downloads/XX-Net-2.9.6/gae_proxy/local/proxy.py", line 95, in <module>
    from cert_util import CertUtil
  File "/Users/apple/Downloads/XX-Net-2.9.6/gae_proxy/local/cert_util.py", line 38, in <module>
    import OpenSSL
  File "/Users/apple/anaconda/lib/python2.7/site-packages/OpenSSL/__init__.py", line 8, in <module>
    from OpenSSL import rand, crypto, SSL
ImportError: dlopen(/Users/apple/anaconda/lib/python2.7/site-packages/OpenSSL/rand.so, 2): Library not loaded: libssl.1.0.0.dylib
  Referenced from: /Users/apple/anaconda/lib/python2.7/site-packages/OpenSSL/rand.so
  Reason: image not found

Traceback (most recent call last):
  File "launcher/start.py", line 135, in <module>
    main()
  File "launcher/start.py", line 110, in main
    module_init.start_all_auto()
  File "/Users/apple/Downloads/XX-Net-2.9.6/launcher/module_init.py", line 98, in start_all_auto
    start(module)
  File "/Users/apple/Downloads/XX-Net-2.9.6/launcher/module_init.py", line 37, in start
    proc_handler[module]["imp"] = __import__(module, globals(), locals(), ['local', 'start'], -1)
  File "/Users/apple/Downloads/XX-Net-2.9.6/gae_proxy/start.py", line 14, in <module>
    import local.proxy as client
  File "/Users/apple/Downloads/XX-Net-2.9.6/gae_proxy/local/proxy.py", line 95, in <module>
    from cert_util import CertUtil
  File "/Users/apple/Downloads/XX-Net-2.9.6/gae_proxy/local/cert_util.py", line 38, in <module>
    import OpenSSL
  File "/Users/apple/anaconda/lib/python2.7/site-packages/OpenSSL/__init__.py", line 8, in <module>
    from OpenSSL import rand, crypto, SSL
ImportError: dlopen(/Users/apple/anaconda/lib/python2.7/site-packages/OpenSSL/rand.so, 2): Library not loaded: libssl.1.0.0.dylib
  Referenced from: /Users/apple/anaconda/lib/python2.7/site-packages/OpenSSL/rand.so
  Reason: image not found
Stopping all modules before exit!
Apr 02 13:51:26.136 - [ERROR] stop module gae_proxy fail:'imp'
Apr 02 13:51:26.136 - [ERROR] Except stack:Traceback (most recent call last):
  File "/Users/apple/Downloads/XX-Net-2.9.6/launcher/module_init.py", line 71, in stop
    _start = proc_handler[module]["imp"].start
KeyError: 'imp'

logout
Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

[进程已完成]

--- a/start.command
+++ b/start.command
@@ -1,2 +1,3 @@
+#!/bin/bash
 cd "$(dirname "$0")"
-/usr/bin/env python2.7 launcher/start.py
+/usr/bin/python2.7 launcher/start.py

